.github/workflows/tag.yml

```diff
@@ -10,6 +10,8 @@ on:
 jobs:
   tag:
     name: Push Git Tag
+    env:

```

Comment:
Can you move this to the step as good practice? So the secret is only available for it

Reply:
Fixed